src/libraries/Microsoft.Extensions.DependencyModel/src/CollectionExtensions.cs

```diff
@@ -35,9 +35,16 @@ public static IEnumerable<string> GetRuntimeAssets(this IEnumerable<RuntimeAsset
 
         private static IEnumerable<string> GetAssets(IEnumerable<RuntimeAssetGroup> groups, string runtime)
         {
-            return groups
-                .Where(a => string.Equals(a.Runtime, runtime, StringComparison.Ordinal))
-                .SelectMany(a => a.AssetPaths);
+            foreach (RuntimeAssetGroup group in groups)
```

Comment:
`GetRuntimeFiles()` below is using a similar expression (selecting `RuntimeFiles` instead of `AssetPaths`).

Reply:
Thanks.  Added.